projects/ngx-fixture/src/lib/matchers.ts

```diff
@@ -0,0 +1,38 @@
+const elementMatchers: jasmine.CustomMatcherFactories = {
+
+  /**
+   * Checks that an element has the specified class
+   */
+  toHaveClass: (util: jasmine.MatchersUtil, customEqualityTesters: Array<jasmine.CustomEqualityTester>): jasmine.CustomMatcher => {
+    return {
+      compare: (el: any, expected: string): jasmine.CustomMatcherResult => {
+        if (!el) {
+          return { pass: false, message: `Expect element to have class '${expected}', but element was null or undefined` };
+        }
+        const actual = el.classes;
+        const pass = actual.includes(expected);
+        const message = `Expected element to have class '${expected}', but had '${actual.join(', ')}'`;
+        return { pass, message };
+      }
+    };
+  },
+
+    /**
+   * Checks that an element has the specified value
+   */
+  toHaveValue: (util: jasmine.MatchersUtil, customEqualityTesters: Array<jasmine.CustomEqualityTester>): jasmine.CustomMatcher => {
+    return {
+      compare: (el: any, expected: string): jasmine.CustomMatcherResult => {
+        if (!el) {
+          return { pass: false, message: `Expect element to have value '${expected}', but element was null or undefined` };
+        }
+        const actual = el.value;
+        const pass = actual === expected;
+        const message = `Expected element to have value '${expected}', but had value '${actual}'`;
+        return { pass, message };
+      }
+    };
+  }
+};
+
+export { elementMatchers };
```

Comment:
testElementMatchers, given that they all apply to instances of TestElement? Or simply customMatchers, but then there might be a name clash with other libs choosing the same generic name. Or speculoosMatchers? 
Or speculoosJasmineMatchers, to make it clear that this is about jasmine, and allow adding matchers for other test frameworks some day?